src/CTA.WebForms2Blazor/Factories/FileConverterFactory.cs

```diff
@@ -36,29 +36,29 @@ public FileInformation Build(FileInfo document)
             string relativePath = Path.GetRelativePath(_sourceProjectPath, document.FullName);
             string extension = document.Extension;
 
-            FileInformation fi;
+            FileConverter fi;
```

Comment:
Rename `fi` to `fc` to match previous naming convention or to `fileConverter` as another suggestion.